pkgs/by-name/he/helm-sops/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  helm,
+  sops,
+}:
+
+buildGoModule rec {
+  pname = "helm-sops";
+  version = "20250205-1";
+  src = fetchFromGitHub {
+    owner = "camptocamp";
+    repo = "helm-sops";
+    rev = "20250205-1";
```

since the project uses tags lets use tags aswell
```
tag = version;
```

---

```suggestion
    tag = version;
```
since the attrset is recursive you can reference it from inside of it.

There has also been a movement towards using `tag` instead of `rev` if you are keying off a git tag